x/icaauth/client/cli/flags.go

```diff
@@ -0,0 +1,19 @@
+package cli
+
+import (
+	flag "github.com/spf13/pflag"
+)
+
+const (
+	// The connection end identifier on the controller chain
+	FlagConnectionID = "connection-id"
+)
+
+// common flagsets to add to various functions
+var (
+	fsConnectionID = flag.NewFlagSet("", flag.ContinueOnError)
+)
+
+func init() {
+	fsConnectionID.String(FlagConnectionID, "", "Connection ID")
+}
```

Comment:
please don't use global variables for this - use the helpers like is done with other flags.

Reply:
ok